ts_src/index.ts

```diff
@@ -185,6 +185,18 @@ export function validateMnemonic(
 export function setDefaultWordlist(language: string): void {
   const result = wordlists[language];
   if (result) DEFAULT_WORDLIST = result;
+  else
+    throw new Error('Could not find wordlist for language "' + language + '"');
+}
+
+export function getDefaultWordlist(): string {
+  if (!DEFAULT_WORDLIST) throw new Error('No Default Wordlist set');
```

Comment:
Maybe this should just return null...?

Reply:
I think `undefined` is fine.

Reply:
Why don't you export `DEFAULT_LANGUAGE` from `_wordlists.ts` and use it?

Reply:
what do you mean?